docs/source/configuration.rst

```diff
@@ -0,0 +1,18 @@
+Configuration
+=============
+
+:attr:`umodbus.conf` is a global configuration object and is an instance of
+`umodbus.config.Config`. This instance can be used like this:
+
+.. code:: python
+    
+  from umodbus import conf
+
+  conf.SIGNED_VALUES = True 
+
+
+.. module:: umodbus.config
+
+.. autoclass:: Config
+    :members:  SIGNED_VALUES
```

Comment:
No, but I understand the confusion. I already included some code for a new feature: 32 bit values. The `BIT_SIZE` settings is related to this issue, but isn't functional yet. Currently bit size is 16 bit always, ignore this setting. That is the reason for not in including this setting in the docs right now.